src/App.vue

```diff
@@ -222,6 +222,9 @@ footer {
     top: 13px;
     font-size: 18px;
 }
+.md-theme-grey.md-input-container.md-input-focused label {
+  color: rgba(0, 0, 0, 0.54) !important;
```

Comment:
can we make more specified selector, add additional class, instead of using important.